src/mscorlib/src/System/Threading/ThreadPool.cs

```diff
@@ -36,6 +36,9 @@ internal static class ThreadPoolGlobals
         public static volatile bool vmTpInitialized;
         public static bool enableWorkerTracking;
 
+        public const int wsqNowActive = 2; // Require at least 2 full observations of empty wsq to transition to inactive
+        public static int wsqActive;
```

Comment:
Oh. `internal static class WorkStealingQueueList` So there is only one class so static member variables are adequate.

So stealing crosses thread pools and appdomains.  Sounds like numa aware will need to fix that.